### PR TITLE
chore: implementing TW's feedback in the comments in Compute Samples

### PR DESCRIPTION
**Pull request by rsamborski**

### CODE REVIEW

File: compute/api/Compute.Samples/CreateInstanceAsync.cs
Comment (amanda-tarafa) [inline compute/api/Compute.Samples/CreateInstanceAsync.cs:25]: This text is not C# idiomatic nor it matches what this code is. It should read something like (Here and everywhere):
```suggestion
        // TODO(developer): Set your own default values for these parameters or pass different values when calling this method.
```

File: compute/api/Compute.Samples/DeleteInstanceAsync.cs
Comment (amanda-tarafa) [inline compute/api/Compute.Samples/DeleteInstanceAsync.cs:35]: In C#, it wouldn't be IntancesClient.close(), it would be client.Dispose(), but none of these client classes are disposable (probably because they don't need to be) so, this part of the comment is not accurate. Here and everywhere.

```suggestion
        // Initialize client that will be used to send requests. This client only needs to be created
        // once, and can be reused for multiple requests.
```

File: compute/api/Compute.Samples/GetUsageExportBucketAsync.cs
Comment (amanda-tarafa) [inline compute/api/Compute.Samples/GetUsageExportBucketAsync.cs:43]: null and empty are two different values in C#, they have the same effect on the prefix. Here and everywhere.
```suggestion
                $"to null or empty values causes the report to have the default prefix of `usage_gce`.");
```

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
Some of the wording here is either not relevatn for C# or incorrectly describes the code.

@jskeet for a second pair of eyes.

File: compute/api/Compute.Samples/GetUsageExportBucketAsync.cs
Comment (jskeet) [reply compute/api/Compute.Samples/GetUsageExportBucketAsync.cs:43]: Possibly "to null or an empty string" - but generally, yes.

**Comment by jskeet** [comment]
I agree with all Amanda's proposals.

**Comment by rsamborski** [comment]
Updated per your recommendations.

**Review by amanda-tarafa — APPROVED** [review]
LGTM

Will wait on tests to go green and merge after.

**Comment by amanda-tarafa** [comment]
Failures are unrelated and being tracked:

- #1311 for SQL
- #1272 for Storage

Force merging.

**Pull request merged**
